demos/apps/bookinfo/cleanup.sh

```diff
@@ -25,4 +25,5 @@ istioctl delete route-rule ratings-test-delay
 #istioctl delete mixer-rule ratings-ratelimit
 
 kubectl delete -f $SCRIPTDIR/bookinfo-istio.yaml
-kubectl delete -f $SCRIPTDIR/../../istio
+kubectl delete -f $SCRIPTDIR/../../ingress-controller.yaml
+kubectl delete -f $SCRIPTDIR/../../../kubernetes/istio-install
```

Comment:
Please remove this, it does not belong to the book info, it belongs in the istio-install directory.

The user deploying the apps may not be the same as the user installing istio or kubernetes.